packages/api/lib/granule-remove-from-cmr.js

```diff
@@ -60,15 +62,19 @@ const unpublishGranule = async (knexOrTransaction, granule) => {
       {
         published: false,
         cmr_link: undefined,
-      }
+      },
+      ['cumulus_id']
     );
   } catch (error) {
     if (!(error instanceof RecordDoesNotExist)) {
       throw error;
     }
   }
 
-  return granuleModelClient.update({ granuleId: granule.granuleId }, { published: false }, ['cmrLink']);
+  return {
+    dynamoGranule: await granuleModelClient.update({ granuleId: granule.granuleId }, { published: false }, ['cmrLink']),
+    pgGranule: pgGranule,
+  };
```

Comment:
pgGranule might be undefined here and that should be expected.

Previously we weren't using the return value so this change is geared specifically for bulk deletes.

Reply:
👍 sorry, I'm psyched to have docstrings on this project but not used to it so I tend to forget 😄 

https://github.com/nasa/cumulus/pull/2108/commits/4b99df2da3ba283bd0b0d8caabdf69d77eafc902